src/main/java/Character.java

```diff
@@ -31,4 +41,26 @@ public boolean removeItem(String item){
         return inventory.remove(item);
     }
 
+    public int getCurrentHealth(){
+        return this.currentHealth;
+    }
+
+    public int getMaxHealth() {
+        return maxHealth;
+    }
+
+    public void setCurrentHealth(int currentHealth) {
```

Comment:
maybe have an adjust currenthealth method as well. (adding or subtracting a value vs. setting it to a value)

Reply:
I was thinking we leave the logic in combat/consumable: e.g. we do the calculation and then just call the method of setting it.